config/kind-ci/imagePullPolicyNever.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
Why do we need this? Under CI I expect the cluster to be created every time

Reply:
in this PR it is not totally necessary since we're using an image that is not tagged latest, but it is something that they recommend just to be sure https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster